modules/spacing.styl

```diff
@@ -1,8 +1,6 @@
+$unit = 1rem
```

Comment:
Not sure why this has escaped the `spacer-maker` function? Worried this might cause issues with it due to scope?

Reply:
Tweaked spacer-maker() to take $unit as a parameter (defaulting to 1rem) so it's a bit more flexible and self-contained. Also wrapped $value * $unit in parentheses to stop Stylus from causing issues.